### PR TITLE
Syndicate borg rework and general fixes

### LINKED ISSUE #1281: Respawn system is dumb - requires body be destroyed
The requirement is silly, and should be removed.

### PR DESCRIPTION
**Pull request by Alberyk**
Changes:
-added some vars that allow for more pre-set borg models, ported from the newest bay changes
-added combat module and an ert borg as admin spawn presets
-fixed syndicate cyborg
-syndicate cyborgs explode when killed now
-added a limited frag grenade launcher to the syndicate module
-fixed a lot of broken guns
-fixed the bad ert id code added by the ert update
-fixed the tactical shield to have sprites and behave properly
-tweaked even more the vending machine at sec armory
-disabled joining as actor
-enabled spess bikes, heisters get two of them and can get one via cargo contraband system
-replaced advanced advanced e-guns with energy rifles
-fixed some missing id sprites
-fix #1281

### CODE REVIEW

File: code/game/antagonist/outsider/ert.dm [hunk 1]
```diff
@@ -39,10 +41,6 @@ var/datum/antagonist/ert/ert
 	player.equip_to_slot_or_del(new /obj/item/clothing/shoes/swat(src), slot_shoes)
 	player.equip_to_slot_or_del(new /obj/item/clothing/gloves/swat(src), slot_gloves)
 	player.equip_to_slot_or_del(new /obj/item/clothing/glasses/sunglasses(src), slot_glasses)
-	player.implant_loyalty(player) //aparantly they are loyalty implanted??
```
Comment (NanakoAC) [inline code/game/antagonist/outsider/ert.dm]: why would ERT not have loyalty implants? They're given military grade weapons and exosuits, plus an all access pass to a high security research station

File: code/modules/mob/living/silicon/robot/robot.dm [hunk 2]
```diff
@@ -73,6 +73,14 @@
 	var/wiresexposed = 0
 	var/locked = 1
 	var/has_power = 1
+
+	var/lawpreset = /datum/ai_laws/nanotrasen
+	var/spawn_module = null
+	var/key_type = null
+	var/spawn_sound = 'sound/voice/liveagain.ogg'
+	var/cell_type = /obj/item/weapon/cell/high
+	var/pitch_toggle = 1
```
Comment (NanakoAC) [inline code/modules/mob/living/silicon/robot/robot.dm]: Whats the point of this var? I don't see anywhere its set to zero.

Personally i like the variable pitch, it adds a lot of variety even to mechanical sounds

File: code/modules/mob/living/silicon/robot/syndicate_robot.dm [hunk 3]
```diff
@@ -7,32 +7,21 @@
 	icon = 'icons/mob/robots.dmi'
 	icon_state = "syndie_bloodhound"
 	lawchannel = "State"
+	lawpreset = /datum/ai_laws/syndicate_override
+	idcard_type = /obj/item/weapon/card/id/syndicate
+	spawn_module = /obj/item/weapon/robot_module/syndicate
+	key_type = /obj/item/device/encryptionkey/syndicate
+	spawn_sound = 'sound/mecha/nominalsyndi.ogg'
```
Comment (NanakoAC) [inline code/modules/mob/living/silicon/robot/syndicate_robot.dm]: good use of audio

File: code/game/antagonist/outsider/ert.dm (hunk 1 above)
Comment (Alberyk) [inline code/game/antagonist/outsider/ert.dm]: Same reason we don't give to sec and the like, also, loyalty implants can get on the way of antag status, aka; being ert is classified as antag.

File: code/modules/mob/living/silicon/robot/robot.dm (hunk 2 above)
Comment (Alberyk) [inline code/modules/mob/living/silicon/robot/robot.dm]: As I said, it can be used in any future cyborg preset someone may use.

**Pull request merged**